tests/test_cli.py

```diff
@@ -163,6 +164,63 @@ def test_validate_command_with_edx_format(event):
     assert event_str in result.output
 
 
+def test_cli_convert_command_usage():
+    """Tests ralph convert command usage."""
+
+    runner = CliRunner()
+    result = runner.invoke(cli, ["convert", "--help"])
+
+    assert result.exit_code == 0
+    assert (
+        "Options:\n"
+        "  From edX to xAPI converter options: \n"
+        "    -u, --uuid-namespace TEXT     The UUID namespace to use for the `ID` field\n"
+        "                                  generation\n"
+        "    -h, --home_page TEXT          The `actor.account.homePage` to use in the\n"
+        "                                  xAPI statements\n"
+        "  -f, --from [edx]                Input events format to convert  [required]\n"
+        "  -t, --to [xapi]                 Output events format  [required]\n"
+        "  -I, --ignore-errors             Continue writing regardless of raised errors\n"
+        "  -F, --fail-on-unknown           Stop converting at first unknown event\n"
+    ) in result.output
+
+    result = runner.invoke(cli, ["convert"])
+    assert result.exit_code > 0
+    assert (
+        "Error: Missing option '-f' / '--from'. Choose from:\n\tedx"
+    ) in result.output
+
+
+@settings(max_examples=1)
+@given(
+    st.builds(UIPageClose, referer=provisional.urls(), page=provisional.urls()),
+)
+@pytest.mark.parametrize("valid_uuid", ["ee241f8b-174f-5bdb-bae9-c09de5fe017f"])
+def test_cli_convert_command_from_edx_to_xapi_format(valid_uuid, event):
+    """Tests the convert command from edx to xapi format."""
+
+    event_str = event.json()
+    runner = CliRunner()
+    result = runner.invoke(
+        cli, ["convert", "-f", "edx", "-t", "xapi", "-u", valid_uuid], input=event_str
+    )
+    assert result.exit_code == 0
+    assert "http://adlnet.gov/expapi/verbs/terminated" in result.output
```

Comment:
Can you load the output in the expected model to validate the output?

Reply:
Great Idea) Thanks) Now validating using the xAPI pydantic model)